code/modules/mob/mob.dm

```diff
@@ -133,6 +133,40 @@
 		else
 			M.show_message(message, 1, blind_message, 2)
 
+//Same as contained_visible_message, except it takes two additional variables

```

Comment:
This entire proc seems like an unnecessary addition when the two new variables can be simply added to visible_message.

Reply:
I didn't really want to fuck anything over. Would it be okay to just modify the already existing variable?

Reply:
You can easily make it so that visible message defaults for these two final variables to default to negative to add reverse compatibility.